services/minespace-web/src/styles/components/SteppedForm.scss

```diff
@@ -30,6 +30,10 @@
   line-height: 25px;
 }
 
+.ant-dropdown-menu-item:hover, .ant-dropdown-menu-item button:hover{
```

Comment:
nitpick- might be more at home with Tables.scss. There's some other styling of the dropdown menu there.